rasa/core/training/interactive.py

```diff
@@ -816,10 +816,15 @@ def _write_stories_to_file(
     else:
         append_write = "w"  # make a new file if not
 
+    with open(
+        export_story_path, "r", encoding=rasa.shared.utils.io.DEFAULT_ENCODING
+    ) as f:
+        i = sum(1 for s in f if 'interactive_story_' in s) + 1  # count the times string 'interactive story' occur in file
```

Comment:
let's make `interactive_story_ ` a constant to ensure it stays the same between this line and L831